src/components/SelectionList/RadioListItem.tsx

```diff
@@ -55,7 +55,8 @@ function RadioListItem({
                             styles.sidebarLinkTextBold,
                             isMultilineSupported ? styles.preWrap : styles.pre,
                             item.alternateText ? styles.mb1 : null,
-                            isDisabled && styles.colorMuted,
+                            /* eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing */
+                            (isDisabled || item.applyDisabledStyle) && styles.colorMuted,
```

Comment:
A little bit confusing logic, can we just change it so that we align with eslint rule
```suggestion
(isDisabled || item.applyDisabledStyle) ? styles.colorMuted : null,
```